minimal-notebook/Dockerfile

```diff
@@ -19,7 +19,6 @@ RUN apt-get update && apt-get install -yq --no-install-recommends \
     libxrender1 \
     lmodern \
     netcat \
-    pandoc \
```

Comment:
I don't see pandoc installed elsewhere in minimal-notebook via conda. I'm guessing one of the conda packages in base-notebook now specifies pandoc as a dependency and so it's getting installed there. I don't see it listed in the recipes for notebook, jupyterlab, or jupyterhub on conda-forge, so I'm a bit confused about where it's getting pulled in. We should probably make it an explicit conda install until we know for sure it's not an accidental dependency.

Do we know if tex still needs to be installed at the OS level or if the pandoc conda package installs everything it needs too?

Reply:
You're right I had not checked. The pandoc dependency comes form nbconvert.

```
nbconvert 5.6.1 py37_0
----------------------
file name   : nbconvert-5.6.1-py37_0.tar.bz2
name        : nbconvert
version     : 5.6.1
build       : py37_0
build number: 0
size        : 487 KB
license     : BSD-3-Clause
subdir      : linux-64
url         : https://conda.anaconda.org/conda-forge/linux-64/nbconvert-5.6.1-py37_0.tar.bz2
md5         : 7b04496e019d1f922f9718882eb94752
timestamp   : 2019-10-25 01:42:31 UTC
constraints : 
  - python_abi * *_cp37m
dependencies: 
  - bleach
  - defusedxml
  - entrypoints >=0.2.2
  - jinja2
  - jupyter_core
  - mistune >=0.8.1
  - nbformat >=4.4
  - pandoc >=1.12.1
  - pandocfilters >=1.4.1
  - pygments
  - python >=3.7,<3.8.0a0
  - testpath
  - traitlets >=4.2
```
The environment file can be seen [in the github](https://github.com/jupyter/nbconvert/blob/0391f83c95d01844a60e1a74ba41a77a0e20a120/docs/environment.yml).

However it's unclear since the nbconvert documentation still ask to install pandoc. I will create an issue in the GitHub to get a confirmation on that.

For tex sorry but I don't know, I will check.

Reply:
For tex the nbconvert installation guide has a [dedicated section for tex](https://nbconvert.readthedocs.io/en/latest/install.html#installing-tex) 

```
sudo apt-get install texlive-xetex texlive-fonts-recommended texlive-generic-recommended
```
In the `base-notebook` the required packages are installed and so IMO they have to be kept.

- `texlive-xetex`
- `texlive-fonts-recommended`
- `texlive-generic-recommended`

These are dependencies of `texlive-xetex`, so they could be removed -- just to be lean, the result will be the same.
- `texlive-latex-base`
- `texlive-latex-extra`

```bash
$ apt-cache depends texlive-xetex
# texlive-xetex
#  Depends: tex-common
#  Depends: texlive-base
#  Depends: texlive-binaries
#  Depends: texlive-latex-base
#  Depends: texlive-latex-extra
[...]
```

And the following extra packages is installed, there necessity could be challenged

- `texlive-fonts-extra`

[Here](https://tex.stackexchange.com/a/504566) is a great answer on SO explaining the content and the dependencies between those packages.

Reply:
According to the previous discussion, I've cleaned the tex installation and written a test to check if nbconvert is working properly without pandoc and without removed tex packages.

Reply:
Awesome. Thank you for doing this extra investigation and updating the PR!